README.md

```
@@ -4,6 +4,8 @@
 **Qudi** (`/kʲɯːdiː/`, Quickly Dependency Injection) is an attribute-based **simple** dependency injection helper library.  
 Explicitly, No assembly scan, AOT friendly, and Visualize registrations.
 
+![](./assets/hero.png)
```

markdownlint-cli2 (0.20.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)